kimera_graph_msgs/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>kimera_graph_msgs</name>
```

Comment:
I assume the duplication here comes from the fact that you need to recompile msgs under ROS2 standards?

Reply:
Yea. Though the .msg files could remain the same (except the std_msg::Header type has deprecated the stamp field), but then the package.xml, Cmake all need to be different.